invenio_rdm_records/data/objecttypes.json

```diff
@@ -0,0 +1,678 @@
+    "datacite": {"general": "Image", "type": "Photo"},
+    "eurepo": "info:eu-repo/semantics/other",
+    "parent": {"$ref": "https://zenodo.org/objecttypes/image"},
+    "openaire": {
+      "resourceType": "0025",
+      "type": "dataset"
+    },
+    "csl": "graphic"
+  },
+  {
+    "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+    "internal_id": "image-other",
+    "id": "https://zenodo.org/objecttypes/image/other#",
+    "title": {
+      "en": "Other"
+    },
+    "title_plural": {
+      "en": "Other"
+    },
+    "schema.org": "https://schema.org/ImageObject",
+    "datacite": {"general": "Image", "type": "Other"},
+    "eurepo": "info:eu-repo/semantics/other",
+    "parent": {"$ref": "https://zenodo.org/objecttypes/image"},
+    "openaire": {
+      "resourceType": "0025",
+      "type": "dataset"
+    },
+    "csl": "graphic"
+  },
+  {
+    "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+    "internal_id": "publication-datamanagementplan",
+    "id": "https://zenodo.org/objecttypes/publication/datamanagementplan#",
+    "title": {
+      "en": "Data management plan"
+    },
+    "title_plural": {
+      "en": "Data management plans"
+    },
+    "schema.org": "https://schema.org/CreativeWork",
+    "datacite": {"general": "Text", "type": "Data Management Plan"},
+    "eurepo": "info:eu-repo/semantics/technicalDocumentation",
+    "parent": {"$ref": "https://zenodo.org/objecttypes/publication"},
+    "openaire": {
+      "resourceType": "0009",
+      "type": "publication"
+    },
+    "csl": "report"
+  },
+  
+  {
+    "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+    "internal_id": "publication-annotationcollection",
+    "id": "https://zenodo.org/objecttypes/publication/annotationcollection#",
+    "title": {
+      "en": "Annotation collection"
+    },
+    "title_plural": {
+      "en": "Annotation collections"
+    },
+    "schema.org": "https://schema.org/Collection",
+    "datacite": {"general": "Collection"},
+    "eurepo": "info:eu-repo/semantics/technicalDocumentation",
+    "parent": {"$ref": "https://zenodo.org/objecttypes/publication"},
+    "openaire": {
+      "resourceType": "0009",
+      "type": "publication"
+    },
+    "csl": "report"
+  },
+  {
+    "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+    "internal_id": "publication-taxonomictreatment",
+    "id": "https://zenodo.org/objecttypes/publication/taxonomictreatment#",
+    "title": {
+      "en": "Taxonomic treatment"
+    },
+    "title_plural": {
+      "en": "Taxonomic treatments"
+    },
+    "schema.org": "https://schema.org/ScholarlyArticle",
+    "datacite": {"general": "Text", "type": "Taxonomic treatment"},
+    "eurepo": "info:eu-repo/semantics/other",
+    "parent": {"$ref": "https://zenodo.org/objecttypes/publication"},
+    "openaire": {
+      "resourceType": "0020",
+      "type": "publication"
+    },
+    "csl": "article"
+  }]
+  
```

Comment:
I personally thing this might get into a configuration option later on. This ones are taken from Zenodo, they are here merely for testing purposes. I will try to move some of them just to the tests.